observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,172 @@
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector-logs-writer
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: otel-collector-logs-writer
+subjects:
+  - kind: ServiceAccount
+    name: otel-collector-deployment
+    namespace: openshift-logging
+----
+
+. Create the YAML file to define the `OpenTelemetryCollector` custom resource (CR).
++
+.Example OpenTelemetryCollector
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: openshift-logging
+spec:
+  serviceAccount: otel-collector-deployment
+  config:
+    extensions:
+      bearertokenauth:
+        filename: "/var/run/secrets/kubernetes.io/serviceaccount/token"
+    receivers:
+      otlp:
+        protocols:
+          grpc: {}
+          http: {}
+    processors:
+      resource:
+        attributes: <1>
+          - key: loki.format <2>
+            action: insert
+            value: json
+          - key:  kubernetes_namespace_name
+            from_attribute: k8s.namespace.name
+            action: upsert
+          - key:  kubernetes_pod_name
+            from_attribute: k8s.pod.name
+            action: upsert
+          - key: kubernetes_container_name
+            from_attribute: k8s.container.name
+            action: upsert
+          - key: log_type
+            value: application
+            action: upsert
+          - key: loki.resource.labels <3>
+            value: log_type, kubernetes_namespace_name, kubernetes_pod_name, kubernetes_container_name
+            action: insert
+      transform:
+        log_statements:
+          - context: log
+            statements:
+              - set(attributes["level"], ConvertCase(severity_text, "lower"))
+
+    exporters:
+      loki:
+        endpoint: https://logging-loki-gateway-http.openshift-logging.svc.cluster.local:8080/api/logs/v1/application/loki/api/v1/push <4>
+        tls:
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+        auth:
+          authenticator: bearertokenauth
+
+    service:
+      extensions: [bearertokenauth] <5>
+      pipelines:
+        logs:
+          receivers: [otlp]
+          processors: [transform, resource]
+          exporters: [loki] <6>
+----
+<1> Resource attributes `kubernetes_namespace_name`, `kubernetes_pod_name`, `kubernetes_container_name`, `log_type` are used by OpenShift console. The Loki exporter considers them as labels if they are present in
+<2> Resource attribute `loki.format` configures the format of Loki logs. Possible values are `json`, `logfmt` and `raw`.
+<3> Resource attribute `loki.resource.labels` configures which resource attributes are considered as Loki labels.
+<4> The Loki exporter that points to the gateway of LokiStack `logging-loki` instance and uses `application` tenant.
+<5> Enables `bearertokenauth` extension that is used by the Loki exporter.
```

Comment:
@pavol, should this "`bearertokenauth` extension" be installed as a prerequisite to this procedure?

Reply:
No the extension is part of the collector